tests/unit/src/main/scala/tests/BaseAmmoniteSuite.scala

```diff
@@ -388,6 +389,9 @@ abstract class BaseAmmoniteSuite(scalaVersion: String)
       """errored.sc:15:25: error: not found: type Fooz
         |val decodedFoo = decode[Fooz](json)
         |                        ^^^^
+        |errored.sc:18:8: error: not found: type Foozz
```

Comment:
Why did the error get duplicated?

Reply:
That's not a duplicate. That's changed by https://github.com/scalameta/metals/pull/3790/commits/59b9b8b0a49dac15c34485bc8ee26fdc8cf1c0d6. Accepting `//` comments in test fixtures slightly changes one of the sources used in an Ammonite test, and makes that (legit) error surface.